server/spec/models/wish_spec.rb

```diff
@@ -3,7 +3,6 @@
 RSpec.describe Wish, type: :model do
 
   let!(:api_user){
-    #puts "hi"
```

Comment:
In this file, I noted a :false which ought to have been a false (Symbol/Boolean difference)

Reply:
Yes there were two in the file, I have changed them to boolean.